ludwig/schema/trainer.py

```diff
@@ -165,12 +165,15 @@ class ECDTrainerConfig(BaseTrainerConfig):
     )
 
     regularization_type: Optional[str] = schema_utils.RegularizerOptions(
-        default="l2", description="Type of regularization."
+        default="l2",
+        description="Type of regularization.",
+        parameter_metadata=TRAINER_METADATA["regularization_type"],
     )
 
-    regularization_lambda: float = schema_utils.FloatRange(
+    regularization_lambda: float = schema_utils.NonNegativeFloat(
         default=0.0,
         min=0,
```

Comment:
I don't think `min` is a property of `NonNegativeFloat`

Reply:
Done.